tests/runtime/resource_aggregates/wasm.rs

```diff
@@ -1,22 +1,42 @@
 wit_bindgen::generate!({
     path: "../../tests/runtime/resource_aggregates",
-    exports: {
-        world: Test,
-        "test:resource-aggregates/test": Test,
-        "test:resource-aggregates/test/thing": MyThing,
-    },
 });
 
+use core::ops::{Deref, DerefMut};
 use test::resource_aggregates::test::{foo as import_foo, Thing};
 
 pub struct Test {}
 
+export!(Test);
+
+impl Deref for exports::test::resource_aggregates::test::Thing {
```

Comment:
Would it make sense to generate `Deref` and `DerefMut` automatically as part of bindings generation?

Reply:
Ideally! Unfortunately though that can't be done because the `Target` of the deref, `MyThing`, isn't known at bindings generation time. 

It's possible that during the `export!` macro we could add these impls, but I was running into some issues doing that with weird "this type isn't public enough" when I thought it was, so I didn't bottom that out much further.

Reply:
Oh I also realized that for `export!` implementing `Deref` doesn't work cross-crate which is also why I didn't include it (as that would define a trait for a foreign type)